ruby/ql/src/queries/security/cwe-352/CSRFProtectionDisabled.qhelp

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE qhelp PUBLIC
+ "-//Semmle//qhelp//EN"
+ "qhelp.dtd">
+<qhelp>
+
+  <overview>
+    <p>
+      Cross-site request forgery (CSRF) is a type of vulnerability in which an
+      attacker  is able to force a user carry out an action that the user did
+      not intend. This may allow the attacker to perform actions on behalf of
+      the targeted user.
```

Comment:
This final sentence feels redundant - aren't we just repeating the previous sentence?

Reply:
Good point, I've cut this sentence.